plugins/ExamplePlugin/src/SumOfExponential.py

```diff
@@ -2,16 +2,10 @@
   Author:  A. Alfonsi
   Date  :  11/17/2017
 """
-from __future__ import division, print_function , unicode_literals, absolute_import
-
-#External Modules---------------------------------------------------------------
 import numpy as np
 import math
-#External Modules End-----------------------------------------------------------
```

Comment:
why you removed this? It because basically a standard. It was a nice way to compartmentalize the "imports" from external libraries and the import from "internal entities". I know it is not in all the modules but we might add it back.

This comment is valid for all the modules you did this.

Reply:
I could definitely see this being a useful teaching tool for non-Python users who were not RAVEN developers; since the python standard defines the order of imports, it seemed a little cluttering for every file to have this when often only 1 or 2 imports were in each "block". I left spaces between each block (std, external, RAVEN, internal to module) to emphasize this organization, though.